src/components/z-info-reveal/styles.css

```diff
@@ -0,0 +1,108 @@
+:host {
```

Comment:
ho l'impressione che manchi l'estensione dell'area tappabile a 40px - quindi l'aggiunta di 4px su tutti i lati del componente, che al momento in stato chiuso è di 32x32px e se aperto su una riga è di X x 32px

Reply:
sistemato!